base_classes/NXtransformations.nxdl.xml

```diff
@@ -176,6 +192,207 @@
 			differences.  Use of ``AXISNAME_end`` is recommended.
 		</doc>
 	</field>
+    <field name="distance" type="NX_FLOAT" units="NX_LENGTH" minOccurs="0">
+      <doc>
+	This is the distance to the previous component in the
+	instrument. 
+      </doc>
+      <attribute name="vector" type="NX_NUMBER">
+	<dimensions rank="1">
+	  <dim index="1" value="3" />
+	</dimensions>
+	<enumeration>
```

Comment:
This attribute is defined as having a number type thus can probably cannot be an enumeration.

Reply:
That's not quite right: arrays are not allowed as enumeration items.